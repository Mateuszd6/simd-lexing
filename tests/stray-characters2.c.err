../tests/stray-characters2.c:3:157: error: Unexpected character in file
