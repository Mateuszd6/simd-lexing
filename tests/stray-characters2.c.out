../tests/stray-characters2.c:1:1: TOK 0 (L = 6): "foobar"
../tests/stray-characters2.c:3:1: TOK 0 (L = 3): "int"
../tests/stray-characters2.c:3:5: TOK 0 (L = 152): "DonaudampfschiffahrtselektrizitatenhauptbetriebswerkbauunterbeamtengesellschaftDonaudampfschifffahrtsgesellschaftskapitanRechtsschutzversicherungsgesell"
